File: python/1 - Iniciante/1037_intervalo.py
N = float(input())
if N >= 0 and N <= 50.0000:
    if N >= 0 and N <= 25.0000:
        print(f'Intervalo [0,25]')
    else:
        print(f'Intervalo (25,50]')
elif N >= 50.00001 and N<= 100.0000:
    if N >= 50.00001 and N <= 75.0000:
        print(f'Intervalo (50,75]')
    else:
        print(f'Intervalo (75,100]')
else:
    print(f'Fora de intervalo')

File: c/c99/1 - Iniciante/1037_intervalo.c
#include <stdio.h>

int main() {
    double N;
    scanf("%lf", &N);
    if (N >= 0 && N <= 25.0000) {
        printf("Intervalo [0,25]\n");
    }
    else if (N > 25.00001 && N <= 50.0000) {
        printf("Intervalo (25,50]\n");
    }
    else if (N > 50.00001 && N <= 75.0000) {
        printf("Intervalo (50,75]\n");
    }
    else if (N > 75.00001 && N <= 100.0000) {
        printf("Intervalo (75,100]\n");
    }
    else {
        printf("Fora de intervalo\n");
    }
    return 0;
}
